### PR TITLE
Added scripts for coverage check support new parameter AutoCompleteLi…

### PR DESCRIPTION
**Pull request by Aleksandriys**
…st at KeyboardProperties struct

### CODE REVIEW

**Comment by Aleksandriys** [comment]
@OVikhrov, @aarotar please review.

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---Verification Criteria---
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua]: @Aleksandriys, replace you "Verification criteria" with requirement summary+description+expected result block due to template

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {"upperbound", ""}
+      }
+    })
+  --mobile side: expect SetGlobalProperties response
+   EXPECT_RESPONSE(cid, { success = false, resultCode = "INVALID_DATA" })
+ --mobile side: expecting OnHashChange notification
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua]: @Aleksandriys, no need to specify comment about waiting OnHashChange if in result you don't expect it: Times(0)

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   EXPECT_NOTIFICATION("OnHashChange") 
+   :Times(0)
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_StringOutLowerBound.lua]: @Aleksandriys, please remove redundant empty lines in script

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+     EXPECT_NOTIFICATION("OnHashChange") 
+     :Times(0)
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua]: @Aleksandriys, please remove redundant empty lines in script

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+     "a"
+     },
+      language = "EN-US",
+      autoCompleteText = "Text_1, Text_2",
+      autoCompleteList = {"   "}
+    }
+   })
+   --mobile side: expect SetGlobalProperties response
+  EXPECT_RESPONSE(cid, { success = false, resultCode = "INVALID_DATA" })
+   --mobile side: expecting OnHashChange notification
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua]: @Aleksandriys, no need to specify comment about waiting OnHashChange if in result you don't expect it: Times(0)

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua [hunk 6]
```diff
@@ -0,0 +1,53 @@
+----Verification criteria:
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_SpacesInValue.lua]: @Aleksandriys, replace you "Verification criteria" with requirement summary+description+expected result block due to template

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+  EXPECT_NOTIFICATION("OnHashChange") 
+   :Times(0)
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua]: @Aleksandriys, please remove redundant empty lines in script

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = 5
+      }
+    })
+--mobile side: expect SetGlobalProperties response
+   EXPECT_RESPONSE(cid, { success = false, resultCode = "INVALID_DATA" })
+   --mobile side: expecting OnHashChange notification
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua]: @Aleksandriys, no need to specify comment about waiting OnHashChange if in result you don't expect it: Times(0)

File: test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua [hunk 9]
```diff
@@ -0,0 +1,52 @@
+---Verification Criteria
```
Comment (aarotar) [inline test_scripts/API/SetGlobalProperties/AutoCompleteList/ATF_SetGlobalProperties_N_AutoCompleteList_InvalidValueType.lua]: @Aleksandriys, replace you "Verification criteria" with requirement summary+description+expected result block due to template

**Pull request closed without merging**